data.sql

```diff
@@ -41,4 +41,40 @@ UPDATE animals SET owners_id = 1 WHERE name = 'Agumon';
 UPDATE animals SET owners_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
 UPDATE animals SET owners_id = 3 WHERE name IN ('Devimon', 'Plantmon');
 UPDATE animals SET owners_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
-UPDATE animals SET owners_id = 5 WHERE name IN ('Angemon', 'Boarmon');
\ No newline at end of file
+UPDATE animals SET owners_id = 5 WHERE name IN ('Angemon', 'Boarmon');
+
+
+-- day 4
+INSERT INTO vets (name, age, date_of_graduaction) VALUES 
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'), 
+('Jack Harkness', 38, '2008-06-08');    
+
+INSERT INTO specializations (species_id, vets_id) VALUES 
+(1, 1), 
+(2, 3), 
+(1, 3), 
+(2, 4);
+
+INSERT INTO visits (animals_id, vets_id, visit_date) VALUES 
+(1, 1, '2020-05-24'),
```

Comment:
[OPTIONAL]

- [ ]  Instead of hard coding the values into visits table I would suggest the dynamic way because What if it was some big data and don't know the ids ?
Hint: This can be achieved by saying:

```sql
INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24')
```